pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: grpc_cronet
 description: Flutter plugin that provides Cronet-based grpc client implementation.
-version: 0.0.6
+version: 0.0.7
```

Comment:
We should update the `grpc` dep below as well; I think we now need at least `3.1.0`.

Reply:
Ah, forgot about that, thanks!